internal/commands/objectstorage/objectstorage.go

```diff
@@ -0,0 +1,21 @@
+package objectstorage
+
+import (
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/commands"
+)
+
+// BaseobjectstorageCommand creates the base "objectstorage" command
+func BaseobjectstorageCommand() commands.Command {
+	return &objectstorageCommand{
+		commands.New("objectstorage", "Manage Managed object storage services"),
+	}
+}
+
+type objectstorageCommand struct {
+	*commands.BaseCommand
+}
+
+// InitCommand implements Command.InitCommand
+func (lb *objectstorageCommand) InitCommand() {
```

Comment:
Could use more generic or object-storage related receiver name 🙈

Reply:
Classic. :smile: Renamed all receivers to `c` per "Command".